### DISCUSSION TITLE
Add Support for 'sass-embedded'

### DISCUSSION THREAD

**Discussion by Pekes317** [vercel/next.js#36160] [Ideas]
### Describe the feature you'd like to request

I would like to be able to use `sass-embedded` to compile my SCSS / SASS files.

### Describe the solution you'd like

It looks like the `sass-loader` started support as of `12.6.0` so it would seem that just updating to that that version of the loader and probably adding it as an optional peer dependency.

### Describe alternatives you've considered

Not sure of other alternatives.

**Comment by jakubriedl** [comment]
This is important because `sass`(dart) package is super slow on bigger codebases and `node-sass` is deprecated. `sass-embedded` is fastest of these 3 and fully supported. 
In our builds we estimate that 70-80% of build time is spent on sass so making it faster is really big deal.

**Comment by joostmeijles** [comment]
Is it possible to use a custom configured `sass-loader` in NextJs config as workaround?

**Reply by joostmeijles** [reply]
Got it working using:
```
webpack(config) {
  ...

  for (const rule of config.module.rules) {
    if (rule.oneOf) {
      for (const oneOfRule of rule.oneOf) {
        if (oneOfRule.use && Array.isArray(oneOfRule.use)) {
          for (const use of oneOfRule.use) {
            if (use.loader && use.loader.includes('sass')) {
              use.loader = require.resolve('sass-loader');
              use.options.implementation = require.resolve('sass-embedded');
            }
          }
        }
      }
    }
  }

  ...
}
```

**Reply by joostmeijles** [reply]
PR to add sass-embedded support: https://github.com/vercel/next.js/pull/64577
